File: Lab1/My2DPoint.cpp
#include "My2DPoint.h"

My2DPoint::My2DPoint(int _x, int _y) {
	x = _x;
	y = _y;
}

void set(int _x, int _y) {
	x = _x;
	y = _y;
}

void move(int _x, int _y) {
	x += _x;
	y += _y;
}

int getX() {
	return x;
}

int getY() {
	return y;
}

File: Lab1/My2DPoint.java
public class My2DPoint {
	int x;
	int y;

	public My2DPoint(int x, int y) {
		this.x = x;
		this.y = y;
	}

	public void set(int x, int y) {
		this.x = x;
		this.y = y;
	}

	public void move(int x, int y) {
		this.x += x;
		this.y += y;
	}

	public int getX() {
		return this.x;
	}

	public int getY() {
		return this.y;
	}

	public String toString() {
		return "(" + this.x + ", " + this.y + ")";
	}
}